mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -24,7 +26,8 @@
 
 
 def extract_k_space_center_and_locations(data_values, samples_locations,
-                                         thr=None, img_shape=None):
+                                         thr=None, img_shape=None,
+                                         isFFT=False):
```

Comment:
`isFFT` is not really respecting snake case, let's stick to it.

Reply:
I also realize `data_values` does not have an updated doc, can you do it?

Reply:
On it